telegram-bot-api/HttpStatConnection.cpp

```diff
@@ -39,7 +39,12 @@ void HttpStatConnection::wakeup() {
   td::HttpHeaderCreator hc;
   hc.init_status_line(200);
   hc.set_keep_alive();
-  hc.set_content_type("application/json");
+  if (!content.empty() && content.data()[0] == '{') {
```

Comment:
Oohh..... that's bad lol

Reply:
(I don't know why GitHub started a review, it's ok)

Reply:
Not so bad, we know that the only way for the content to start with `{` is if it is a json

Reply:
you should pass the content-type as an optional parameter of this function

Reply:
you should save "as_json" as an attribute and use it in this condition